code/newapp.js

```diff
@@ -0,0 +1,95 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const chat = document.getElementById('chat');
+  const inputWrapper = document.getElementById('input-wrapper');
+  const nameInput = document.getElementById('name-input');
+  const sendButton = document.querySelector('.send-btn');
```

Comment:
Is there a reason why you chose querySelector here instead of getElementbyId? It works fine, we were just wondering. :)